tests/providers/test_bank.py

```diff
@@ -23,12 +24,19 @@ def is_valid_iban(iban):
     return check % 97 == 1
 
 
-def is_valid_aba(aba):
```

Comment:
This function is used by other tests. We should keep it

Reply:
thanks for the information. Updated.